CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CanHazFunny;
+
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private JokeService? _JokeIn; 
+        private JokeWriter? _JokeOut;
+
+        public JokeService JokeIn { 
+            get {
+                return _JokeIn!;
+            }
+            set {
+                _JokeIn = value ?? throw new ArgumentNullException(nameof(value));
+            } 
+        }
+        public JokeWriter JokeOut
+        {
+            get
+            {
+                return _JokeOut!;
+            }
+            set
+            {
+                _JokeOut = value ?? throw new ArgumentNullException(nameof(value));
+            }
+        }
+
+        public Jester(JokeService jokeIn, JokeWriter jokeOut) 
```

Comment:
```suggestion
        public Jester(IJokeService jokeService, IJokeWriter jokeWriter) 
```
These should be your interfaces. Also, keep the naming the same, just vary the casing throughout your code.